brave/brave-core
# Move `Appearance > Content` settings to the top level of settings menu

As The `Content` setting was hard to find, this PR moves the `Content` settings to the top level of the settings menu. At the same time, moves `Playlist` setting into the `Content > Playlist` like `Content > Speedreader`. These two features are related to contents that're displayed, so it makes sense to group them together.


Resolves https://github.com/brave/brave-browser/issues/36571



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
* Enable Playlist feature flag
* Go to brave://settings
* "Content" should be in the sidebar
* Click the "Content" and see if the page looks like following screenshot

<img width="977" alt="image" src="https://github.com/brave/brave-core/assets/5474642/d765ff30-2eeb-40f9-ae14-e4a766822269">

* In case that flag for Playlist or Speedreader is disabled, each section for them should be hidden
